### PR TITLE
added sphinx documentation

### PR DESCRIPTION
**Pull request by fccoelho**
basic setup. according to issue #56

## Type of Change

- [x ] 📚 Examples / docs / tutorials / dependencies update
- [ ] 🔧 Bug fix (non-breaking change which fixes an issue)
- [ ] 🥂 Improvement (non-breaking change which improves an existing feature)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🔐 Security fix

### CODE REVIEW

**Comment by xmnlab** [comment]
@fccoelho where would this be published? readthedocs? github pages?

**Comment by fccoelho** [comment]
This will be improved continuously, but all the initial bits and pieces are already in place.

**Comment by fccoelho** [comment]
> @fccoelho where would this be published? readthedocs? github pages?

I have been using readthedocs

File: docs/source/overview.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Overview
+The epigraphhub library is designed to provide for external users all the analytical functionality used in powering the EpigraphHub platform and applications. In fact all application designed to run on the EpigraphHub platform rely strongly on it.
```
Comment (xmnlab) [inline docs/source/overview.md:2]:
```suggestion
The epigraphhub library is designed to provide external users with all of the analytical functionality used to power the EpigraphHub platform and applications. In fact, all applications designed to run on the EpigraphHub platform rely heavily on it.
```

File: docs/source/overview.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+This library should be used by users that want to contribute to the platform as well as by users that want the borrow some of the tools developed by our team in other opensource projects.
```
Comment (xmnlab) [inline docs/source/overview.md:4]:
```suggestion
This library should be used by users that want to contribute to the platform, as well as users who want the borrow some of the tools developed by our team in other open source projects.
```

**Review by xmnlab — COMMENTED** [review]
In general, LGTM! just added some comments about text (btw I am not an expert in the English language).

File: docs/source/overview.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Overview
+The epigraphhub library is designed to provide external users with all of the analytical functionality used to power the EpigraphHub platform and applications. In fact, all applications designed to run on the EpigraphHub platform rely heavily on it.
+
+This library should be used by users that want to contribute to the platform as well as by users that want the borrow some of the tools developed by our team in other opensource projects.
```
Comment (eduardocorrearaujo) [inline docs/source/overview.md:4]:
```suggestion
This library should be used by users who want to contribute to the platform and users who want to borrow some of the tools developed by our team in other open-source projects.
```

**Review by eduardocorrearaujo — CHANGES_REQUESTED** [review]

**Pull request merged**